app/src/main/java/com/kamron/pogoiv/widgets/PokemonSpinnerAdapter.java

```diff
@@ -45,19 +45,26 @@ public void updatePokemonList(ArrayList<Pokemon> list) {
 
     }
 
-    /*
+
     @Override
     public View getView(int position, View convertView, ViewGroup parent) {
-        return getCustomView(position, convertView, parent);
+        LayoutInflater inflater = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+
+        TextView row = (TextView) inflater.inflate(textViewResourceId, parent, false);
+        Pokemon pokemon = pokemons.get(position);
+
+        row.setText(pokemon.getDisplayName());
```

Comment:
We'd need to add a comment explaining the whole point of this method is to show `pokemon.getDisplayName()` instead of `pokemon.toString()` (that is, the name). I also wonder whether we should just change toString instead.